tests/api_tests/nodes/test_views.py

```diff
@@ -16,7 +17,7 @@ def setUp(self):
         self.user.set_password('justapoorboy')
         self.user.save()
         self.basic_auth = (self.user.username, 'justapoorboy')
-        self.url = '/v2/'
+        self.url = '/' + API_BASE
```

Comment:
This will be a general change, but for strings, we prefer to manipulate them using format(). https://cosdev.readthedocs.org/en/latest/style_guides/python.html#string-formatting. Other than that it looks good.